modules/displaybook.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-mutable-exports */
+const bookContainer = document.querySelector('.books-container');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+
+let bookList = [];
+
+const bookUpload = () => {
+  bookContainer.innerHTML = '';
+  for (let i = 0; i < bookList.length; i += 1) {
+    bookContainer.innerHTML += `
```

Comment:
**[Optional]**

- [ ] Kindly use `forEach` instead of `for loop` to follow ES6 😊